api/src/test/java/io/opentelemetry/OpenTelemetryTest.java

```diff
@@ -50,7 +50,7 @@ public void before() {
 
   @Test
   public void testDefault() {
-    assertThat(OpenTelemetry.getTracer()).isEqualTo(Trace.getTracer());
+    assertThat(OpenTelemetry.getTracer()).isInstanceOf(NoopTrace.newNoopTracer().getClass());
```

Comment:
isInstanceOf(NoopTrace.class)?

Reply:
No, the tracer impl is private member of `NoopTrace`